rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommandTest.java

```diff
@@ -180,25 +180,10 @@ public void testUnreachableShortURLProvided() throws Exception {
 
     }
 
-    @Test(expected = CLIException.class)
-    public void testInvalidPackageWebbUrlProvided() throws Exception {
-
-        packagePath = "http://www.lamsade.dauphine.fr/~cornaz/Enseignement/ALGO-JAVA/";
-
-        InstallPackageCommand installPackageCommand = new InstallPackageCommand(packagePath);
-        installPackageCommand.execute(currentContextMock);
-        assertThat(stack.get(0).getMessage(),
-                   is("'" + packagePath +
-                      "' is not a valid URL of a proactive package as it does not contain the METADATA.json and the resources folder required for installation."));
-
-        throw stack.get(0);
-
-    }
-
     @Test(expected = CLIException.class)
     public void testInvalidGithubUrlProvided() throws Exception {
 
-        packagePath = "https://github.com/Aminelouati/wrong_github_url/archive/master.zip";
+        packagePath = "https://github.com/ow2-proactive/wrong_github_url/archive/master.zip";
```

Comment:
nice one!

Reply:
it was just a test Luis.... we can put https://github.com/pabloEscobar/

Reply:
I don't want to dig into that repository... ;)